indico/modules/categories/models/categories.py

```diff
@@ -595,6 +605,14 @@ def logo_url(self):
         """Get the HTTP URL of the logo."""
         return url_for('categories.display_logo', self, slug=self.logo_metadata['hash'])
 
+    @property
+    def i18n_title(self):
+        return self.title_translations.get(session.lang, self.title)
+
+    @property
+    def i18n_description(self):
+        return self.description_translations.get(session.lang, self.description)
```

Comment:
`description` has some special handling for rendering html/markdown (see `DescriptionMixin`); this logic needs to be preserved here. See RenderModeMixin `_render_getter` on what you need there.